FWCore/Framework/test/test_non_event_ordering_beginLumi_cfg.py

```diff
@@ -14,4 +14,5 @@
                            consumesBeginLuminosityBlock = cms.InputTag("a", "beginLumi"),
                            expectBeginLuminosityBlock = cms.untracked.int32(1))
 
-process.schedule = cms.Schedule(tasks=cms.Task(process.a,process.b,process.c))
+process.t = cms.Task(process.a, process.c)
+process.p = cms.Path(process.b, process.t)
```

Comment:
Why force the task on the Path rather than just the Schedule?

Reply:
Didn't come to my mind. Combined with the previous question, are you perhaps thinking 
```python
process.p = cms.Path(process.b)
process.schedule = cms.Schedule(process.p, tasks=cms.Task(process.a, process.c))
```
?

Reply:
My question is why did you bother to change what was already there?

Reply:
If all the modules are in Tasks they will all be deleted. I need the last one of the consumer chain to be scheduled to keep the rest alive.

Reply:
I had missed the fact that you had moved one of the modules to a Path.

Reply:
I noticed this empirically while doing my ProcessBlock testing. It is not at all obvious looking at it, but this change just coincidentally happens to give the execution order 'a' then 'c' then 'b'. Since the purpose of the test is to demonstrate the Framework is forcing that execution order we should probably perturb this in some way so it isn't just doing it by accident. Renaming 'a' to 'd' seems to do the trick. Without looking at the internals of code, it appears to be running the Tasks alphabetically first (I'm just guessing).

Reply:
Good catch David, thanks. Is the test good-enough for ProcessBlock purposes? I.e. would it be sufficient to rename `a` to `d` in this PR?

Reply:
Yes. At the moment in my branch, I added two new cfg.py's that do the analogous tests for beginProcessBlock and endProcessBlock with 'a' renamed to 'd'.

Reply:
Renamed `a` to `d` in all four non-event transition tests.